articles/active-directory/saas-apps/lessonly-tutorial.md

```diff
@@ -121,17 +121,17 @@ In this section, you enable Azure AD single sign-on in the Azure portal and conf
 	b. In the **Identifier** textbox, type a URL using the following pattern:
 	| |
 	|--|
-	| `https://<companyname>.lesson.ly/auth/saml/metadata`|
+	| `https://<companyname>.lessonly.com/auth/saml/metadata`|
 	| `https://<companyname>.lessonly.com/auth/saml/metadata`|
 
 	> [!NOTE] 
-	> These values are not real. Update these values with the actual Sign-On URL and Identifier. Contact [Lesson.ly Client support team](mailto:dev@lessonly.com) to get these values. 
+	> These values are not real. Update these values with the actual Sign-On URL and Identifier. Contact [Lessonly.com Client support team](mailto:dev@lessonly.com) to get these values. 
 
 1. On the **SAML Signing Certificate** section, click **Certificate(Base64)** and then save the certificate file on your computer.
 
-	![Configure Single Sign-On](./media/lessonly-tutorial/tutorial_lesson.ly_certificate.png)
+	![Configure Single Sign-On](./media/lessonly-tutorial/tutorial_lessonly.com_certificate.png)
```

Comment:
@susanesho This photo still appears to be broken. I believe we need to revert this back to what it originally was 👍